lib/ansible/module_utils/network/nxos/config/lldp_global/lldp_global.py

```diff
@@ -0,0 +1,244 @@
+            commands = self._state_replaced(want, have)
+        return commands
+
+    def _state_replaced(self, want, have):
+        """ The command generator when state is replaced
+
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = ''
+        diff = dict_diff(have, want)  # diff will contain unique and new values in want
+        wantk = want.keys()  # find keys in want to ignore them for defaults
+        havek = have.keys()
+        wantk = self.get_nested_keys(want, wantk)
+        havek = self.get_nested_keys(have, havek)
+        include_params = list(set(havek) - set(wantk))  # keys in have and not in want; they need to be defaulted
+        for param in include_params:
+            commands += 'lldp '
+            if 'holdtime' in param:
+                commands += 'holdtime 120\n'
+            elif 'timer' in param:
+                commands += 'timer 30\n'
+            elif 'reinit' in param:
+                commands += 'reinit 2\n'
+            elif 'port_id' in param:
+                commands += 'portid-subtype 0\n'
+            else:
+                commands += param + '\n'
+        commands += self.set_commands(diff)
+        return commands
+
+    def get_nested_keys(self, dict_param, keys):
+        """
+        Returns names of keys nested inside tlv_select
+        """
+        if 'tlv_select' in keys:
+            keys.remove('tlv_select')
+            for key in dict_param['tlv_select'].keys():
+                if not isinstance(dict_param['tlv_select'][key], dict):
+                    x = 'tlv_select ' + key
+                    x = x.replace('_', '-')
+                    keys.append(x)  # tlv-select dcbxp, tlv-select power-management
+                else:
+                    for k1 in dict_param['tlv_select'][key].keys():
+                        key = key.replace('_', '-')
+                        if k1 == 'v4' or k1 == 'v6':
+                            x = 'tlv-select ' + key + ' ' + k1
+                        else:
+                            x = 'tlv-select ' + key + '-' + k1
+                        keys.append(x)
+        return keys
+
+    def _state_merged(self, want, have):
+        """ The command generator when state is merged
+
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = ''
+        diff = dict_diff(have, want)
+        commands += self.set_commands(diff)
+        return commands
+
+    def _state_deleted(self, have):
+        """ The command generator when state is deleted
+
+        :rtype: A list
+        :returns: the commands necessary to remove the current configuration
+                  of the provided objects
+        """
+        commands = ''
+        for key, val in have.items():
+            if 'holdtime' in key:
```

Comment:
set the parameters to `default` by prepending `no`, not appending default values.